_clients/java.md

```diff
@@ -90,8 +93,9 @@ static class IndexData {
   }
 }
 ```
+{% include copy.html %}
 
-## Initialize the client with SSL and TLS enabled
+## Initializing the client with SSL and TLS enabled
 
 This code example uses basic credentials that come with the default OpenSearch configuration. If you’re using the Java client with your own OpenSearch cluster, be sure to change the code to use your own credentials.
```

Comment:
I think this flows a little better. 

```suggestion
This code example uses basic credentials that come with the default OpenSearch configuration. If you’re using the Java client with your own OpenSearch cluster, be sure to change the code and use your own credentials.
```

Reply:
I will defer to @natebower  because the meaning is that you should change the code so the code uses your own credentials. I think "to" conveys the meaning a little better.